lib/game.rb

```diff
@@ -0,0 +1,47 @@
+require_relative '../lib/game_state'
+require_relative '../lib/game_presenter'
+require_relative '../lib/messages'
+require_relative '../lib/game_settings'
+
+class Game
+    include Validator
+    include Messages
+
+    attr_accessor :game_state, :game_presenter
+
+    def initialize(game_presenter: ConsoleIO.new)
+        @game_presenter = game_presenter
+    end
+
+    def set_game_settings(settings:)
+        settings.reduce({}) do |map, setting_type|
+            input = nil
+            while !setting_type.valid?(input: input)
+                print_banner
+                game_presenter.output_message(setting_type.message)
```

Comment:
print_banner and game_presenter output message, should all be bundled into something like `output_game_settings` and placed into GamePresenter??